grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -1764,7 +1764,7 @@ class JsonController {
         ProductSupplier productSupplier = ProductSupplier.findById(params.productSupplierId)
         ProductPackage productPackage = productSupplier?.defaultProductPackage
         render([
-                unitPrice: productPackage?.price ? g.formatNumber(number: productPackage?.price) : null,
+                unitPrice: productPackage?.productPrice?.price ? g.formatNumber(number: productPackage?.productPrice?.price) : null,
```

Comment:
Do we actually need to make the change to productPackage.price now? I don't see a requirement for this in the ticket.